### ISSUE TITLE
Pseudo-Chilblains on Fingers

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7116]
**Preferred term label:**
Pseudo-chilblains on fingers

**Synonyms** _(updated 2021-09-13)_
chilblains-like fingers
chilblains-like finger lesions

**layperson terms** _(updated 2021-09-13)_
COVID fingers
COVID finger

**Definition (free text, please give PubMed ID)**
Inflammatory chilblain-like nodules on the hands and/or fingers. [PMID33972057]

**Parent term (use hpo.jax.org/app)**
HP:0033696 - Pseudo-chilblain

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-19

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

**Comment by pnrobinson** [comment]
added, thanks
